app/code/Magento/InventoryInStorePickupAdminUi/Controller/Adminhtml/Order/NotifyPickup.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupAdminUi\Controller\Adminhtml\Order;
+
+use Exception;
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\InputException;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\InventoryInStorePickupApi\Api\NotifyOrderIsReadyForPickupInterface;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class NotifyPickup
+ *
+ * @package Magento\InventoryInStorePickupAdminUi\Controller\Adminhtml\Order
+ */
+class NotifyPickup extends Action
```

Comment:
Name is slightly misleading. It's notify and ship in reality

Reply:
But...
![image](https://user-images.githubusercontent.com/26086561/55966154-8d6fe280-5c80-11e9-86ac-45d175093104.png)